.github/workflows/e2e-private-cluster.yaml

```diff
@@ -0,0 +1,39 @@
+name: Run e2e private cluster
+on:
+  workflow_dispatch:
```

Comment:
do we want to set this up on pr? We can make it so a member of our team has to be the one to run

Reply:
I tried it but ran into the same issue where github secrets can't be accessed for PRs across forks, so az login fails on PRs